### DISCUSSION TITLE
Vtest through python

### DISCUSSION THREAD

**Discussion by Srikakumanu** [chaitu-ycr/py_canoe#47] [Ideas]
Do we have python package to run vtest test module ?

**Comment by chaitu-ycr** [comment]
hi @Srikakumanu 

executing vtest test configuration is available in py-canoe version 26.1.0
https://chaitu-ycr.github.io/[py-canoe](https://chaitu-ycr.github.io/py-canoe/#execute-test-configuration-test-units)/#execute-test-configuration-test-units
